sdk/python/.pylintrc

```diff
@@ -68,6 +68,7 @@ disable=raw-checker-failed,
         useless-suppression,
         deprecated-pragma,
         use-symbolic-message-instead,
+        method-hidden,
```

Comment:
Bringing in annotations ends up throwing linting errors about hiding methods (which is a lie!  Pylint just isn't smart enough to realize the attributes set in the constructor _ARE_ the methods being "hidden")